test/config.jl

```diff
@@ -27,3 +27,32 @@ include(joinpath("..", "perf", "common.jl"))
     @test config.parsed_args["initial_condition"] == "Bomex"
     @test config.parsed_args["regression_test"] == false
 end
+
+function extract_job_ids(folder_path)
+    job_id_dict = Dict()
+    for (root, _, files) in walkdir(folder_path)
+        for file in files
+            filepath = joinpath(root, file)
+            data = YAML.load_file(filepath)
+            if haskey(data, "job_id")
+                job_id_dict[filepath] = data["job_id"]
+            end
+        end
+    end
+    return job_id_dict
+end
+
+file_to_job_id = extract_job_ids("config")
```

Comment:
I kind of expected that, somewhere, we'd use / test with `Base.allunique`

Reply:
Can this be simplified?

Reply:
This PR is overall a huge upgrade, so I'm going to enable merge now, but maybe we can clean this up later.

Reply:
I will need to update RRTMGP and Climaparameters anyway, I can do it then.